<ui version="4.0" >
 <class>SoundUserConfigBase</class>
 <widget class="QWidget" name="SoundUserConfigBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string></string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QCheckBox" name="chkDisable" >
     <property name="text" >
      <string>&amp;Disable all sounds</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkActive" >
     <property name="text" >
      <string>No sound if windows has &amp;focus</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ListView" native="1" name="lstSound" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListView</class>
   <extends>QWidget</extends>
   <header>listview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chkDisable</tabstop>
  <tabstop>chkActive</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
